client/src/containers/FormBuilderLoader/FormBuilderLoader.js

```diff
@@ -276,6 +256,13 @@ class FormBuilderLoader extends Component {
             this.props.identifier
           );
 
+          const schemaData = formSchema.schema || this.props.schema.schema;
+          const formData = schemaFieldValues(schemaData, overriddenSchema.state);
+
+          // need to initialize the form again in case it was loaded before
+          // this will re-trigger Injector.form APIs, reset values and reset pristine state as well
+          this.props.actions.reduxForm.initialize(this.props.identifier, formData);
+
```

Comment:
also, as an aside note, what is with `redux-form` always having these magical hidden parameters?!?